src/features/ticket/model/ticketOption.ts

```
@@ -0,0 +1,58 @@
+  id: number;
+  name: string;
+  description: string;
+  type: 'SINGLE' | 'MULTIPLE' | 'TEXT';
+  isMandatory: boolean;
+  choices: {
+    id: number;
+    name: string;
+  }[];
+}
+
+export interface TicketOptionResponse extends ApiResponse<TicketOptionsType> {}
```

ESLint

[error] An interface declaring no members is equivalent to its supertype.

(@typescript-eslint/no-empty-object-type)